pkg/operator/controller.go

```diff
@@ -263,8 +268,12 @@ func (c *Controller) handleWork(getter runtimeObjectGetter, o interface{}) error
 		obj, err := getter.Get(c, key)
 		if err != nil {
 			// see if this is a operator bootstrap scenario
-			if kerrors.IsNotFound(err) && key == sampopapi.ConfigName {
-				return c.Bootstrap()
+			if kerrors.IsNotFound(err) {
+				if key == sampopapi.ConfigName {
+					return c.Bootstrap()
+				}
+				logrus.Printf("handleWork resource %s has since been deleted, ignore update event", key)
```

Comment:
It means it in my mind ... not sure what you mean "a name we don't care about"

Perhaps if I detailed the background, it would help us get to common ground:
- this path is the update/create path
- in various testing runs I would see instances where we update a sample, and then a delete it very soon after
- this code path would get the update event, and then when we refetch, the delete had gone through, and we get the non found
- requeueing the update on a not found error serves no purpose, since it is truly gone, so we got in a always repeating requeue cycle ... this prevented it

Alternatives to the wording:
- `this upsert event for resource %s will be ignored since the resource has been deleted since this event was generated`

Or does this elaboration result in an alternative you want to provide?

thanks

Reply:
this specific block deals w/ handling an event for an update to the samples operator resource (not an imagestream or template), right?

Reply:
the `getter` is a locally defined interface that has impls for all 4 api types the operator watches

Reply:
so if i name an imagestream "instance" and then delete it, i can potentially send the system through c.Bootstrap()?

Reply:
dah .. good point ... I'll update to check to getter type as well

Reply:
new commit pushed for that check